### PR TITLE
Feature html color

### PR DESCRIPTION
**Pull request by jlannoy**
It was noticed it in the comment of the private tag() method, but not
really possible to do. Now it is ;)

### CODE REVIEW

File: src/main/java/net/sargue/mailgun/content/Builder.java [hunk 1]
```diff
@@ -371,6 +371,18 @@ public Builder strong(String content) {
         return tag("strong").text(content).end();
     }
 
+    /**
+     * Adds a {@code <span>} block with text content of the required color. The plain text
+     * version will get only the content.
+     *
+     * @param content the text content
+     * @param content the text content
```
Comment (sargue) [inline src/main/java/net/sargue/mailgun/content/Builder.java]: Duplicated @param content, should be color.

**Comment by jlannoy** [comment]
Sorry, I duplicated the param but forgot to change it... Done.

**Comment by sargue** [comment]
I'm having a second though about this. Reviewing the Builder class I see there is no other method like the one you propose. It seems a bit ad-hoc to me.

Maybe a better solution would be just to make the `tag` method public.

**Comment by sargue** [comment]
I'm publishing the `tag` method. See this: https://github.com/sargue/mailgun/wiki/Mail-content-using-content-helpers#low-level-html

Do you think that's enough?

**Comment by jlannoy** [comment]
I completely agree with you. I didn't want to break the public/private scope of your methods but it's more efficient in this way.

Thank you.

**Comment by sargue** [comment]
Great. I'm closing this PR and publishing a new version.

**Pull request closed without merging**